app/views/world_locations/show.html.erb

```diff
@@ -1,142 +0,0 @@
-<% page_title @world_location.world_location_news.title, "UK and the world" %>
```

Comment:
Just want to make sure that we're not going to effect things like https://www.gov.uk/world/uk-mission-to-the-united-nations/ which aren't news pages. Should make sure we test these at release or decide to remove them in favour of the news pages currently rendered by collections (Bruce in conversation on Thurs)

Reply:
This PR is dependent on us solving the issues with International Delegation pages.  So we'll only be able to merge this PR once they are migrated to use Collections